ticket-booker/src/components/Navbar.style.js

```diff
@@ -0,0 +1,131 @@
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+export const NavbarContainer = styled.nav`
+  width: 95%;
+  height: 80px;
+  background-color: #222222;
+  height: ${(props) => (props.extendNavbar ? '100vh' : '80px')};
+  display: flex;
+  flex-direction: column;
+  border-bottom: 1px solid #dbdbdb;
+
+  @media (min-width: 556px) {
+    //if the width is atl
+    height: 80px;
+  }
+`;
+
+export const LeftContainer = styled.div`
+  display: flex;
+  flex: 5%;
+  height: 80px;
+  align-items: center;
+  justify-content: center;
+  /* background: green; */
+`;
+
+export const LogoContainer = styled.div`
```

Comment:
Does each section of the navbar need to be wrapped in two divs as there is some duplicated code between the containers?

Reply:
removed the unnecessary containers;  logocontainer and profilephoto containers